libs/perun/utils/src/lib/perun-utils.ts

```diff
@@ -864,3 +865,33 @@ export function isMemberIndirectString(member: RichMember): string {
   }
   return 'UNKNOWN';
 }
+
+export function escapeDangerousHtml(html: string): string {
+  const config = {
+    FORBID_TAGS: ['script', 'link', 'iframe', 'object', 'embed', 'frame', 'frameset', 'style'],
```

Comment:
Use whitelists instead of blacklists https://stackoverflow.com/questions/2475784/why-use-a-whitelist-for-html-sanitizing